fe/app/components/Quote/QuoteSlider.tsx

```diff
@@ -0,0 +1,17 @@
+import { QuoteSliderProps } from "./Quote.models";
+import QuoteElement from "./QuoteElement";
+
+const QuoteSlider = ({ elements }: QuoteSliderProps) => {
+  return (
+    <>
+      <div className="flex flex-no-wrap overflow-x-scroll test items-start mb-8 gap-x-[58px]">
+        {elements.map((e, i) => (
```

Comment:
For better context, could you rename the `elements` constant to `quotes` and the `e` to `quote`?

Reply:
done!